### DISCUSSION TITLE
Duda de como se usa

### DISCUSSION THREAD

**Discussion by ghost** [UltimaHoarder/UltimaScraper#1139] [Q&A]
Alguien sabe como se utiliza esto? Soy nuevooo

**Comment by LunarPenguin** [comment]
> Hello, i have translated your question to english with Google translate.

Hola, he traducido tu pregunta al inglés con el traductor de Google.

> If you want to know how to use the software i suggest you read the `README.MD` file on the homepage of this repo

Si desea saber cómo usar el software, le sugiero que lea el archivo `README.MD` en la página de inicio de este repositorio.

> if you dont speak english i would suggest translating it using Google translate or asking someone to translate it.

Si no habla inglés, le sugiero que lo traduzca con el traductor de Google o que le pida a alguien que lo traduzca.
